main.go

```diff
@@ -25,13 +25,13 @@ func getFiles() (codem, charm io.ReadCloser) {
 	if *initialCode != "" {
 		codem, err = os.Open(*initialCode)
 		if err != nil {
-			log.Printf("error opening %s: %v\n", *initialCode, err.Error())
+			log.Printf("error opening %s: %v\n", *initialCode, err.Error())// TODO: log -> dialog/logFile
```

Comment:
Acredito que o ideal nesse caso é algum tipo de popup que saia da aplicação quando fechado. Consigo fazer isso, já fiz algo parecido em outra aplicação que usa fyne.

Reply:
Eu tinha pensado em criar uma janela Pai só para exibir os dialogs e fechar tudo quando o dialog fosse fechado. Mas n achei muito elegante essa solução.

Reply:
Vou criar outra issue para resolver especificamente isso, se nenhuma solução mais elegante for encontrada fazemos isso que você mencionou, usando um popup simples wrapped em uma função independente do resto do código da UI.